src/LibraryManager.Vsix/Shared/VsHelpers.cs

```diff
@@ -82,10 +80,10 @@ public static void AddFileToProject(this Project project, string file, string it
 
         public static void AddFilesToProject(this Project project, IEnumerable<string> files)
         {
-            if (project == null || project.IsKind(ProjectTypes.ASPNET_5, ProjectTypes.DOTNET_Core, ProjectTypes.SSDT))
+            if (project == null || IsCapabilityMatch(project, Constants.DotNetCoreWebCapability))
```

Comment:
See my comment above. This doesn't seem right to me, as it will exclude *all* .Net Core Web Projects, right?

Reply:
Added automatically to .net core, my bad!